R/install.R

```diff
@@ -590,6 +600,36 @@ renv_install_package_impl <- function(record, quiet = TRUE) {
 
 }
 
+renv_install_test <- function(package) {
+
+  # make sure we use the current library paths in the launched process
+  rlibs <- paste(renv_libpaths_all(), collapse = .Platform$path.sep)
+  renv_scope_envvars(R_LIBS = rlibs, R_LIBS_USER = "NULL", R_LIBS_SITE = "NULL")
+
+  # make sure R_TESTS is unset here, just in case
+  # https://github.com/wch/r-source/blob/1c0a2dc8ce6c05f68e1959ffbe6318a309277df3/src/library/tools/R/install.R#L76-L79
+  renv_scope_envvars(R_TESTS = NULL)
+
+  # the actual code we'll run in the other process
+  code <- substitute({
+    setTimeLimit(60)
```

Comment:
Or should we just rely on R's own timeout facilities? I think that's only available in recent-ish versions of R.